strings.json

```diff
@@ -5440,7 +5440,7 @@
     "en": "Book Value",
     "de": "Buchwert",
     "tr": "Defter Değeri",
-    "zh-Hant-TW": "賬面價值"
+    "zh-Hant-TW": "淨值 (股東權益)"
```

Comment:
How about '帳面值'?

Reply:
"賬面價值" is the accounting word.
In investor stock conversation, usually the "股東權益" or "淨值" are mentioned instead of "帳面價值", like 某某股票 "股價低於淨值" (you may google that), which I think might be better than accounting word.
Still same, its totally ok to fix up to you. :)

Reply:
Ok got it, I'll change it since I'm not a native speaker and I think I used a translation from the web